amf-api-contract/shared/src/main/scala/amf/apicontract/internal/metamodel/domain/ParameterModel.scala

```diff
@@ -83,7 +84,8 @@ object ParameterModel extends AbstractParameterModel with OptionalField with Exa
       Schema,
       Payloads,
       Examples,
-      Default
+      Default,
+      FederationMetadata,
```

Comment:
You should override the federation metadata field since its type is FederationMetadata when it should be ParameterFederationMetadata